src/Arcus.Observability.Tests.Integration/Serilog/Sinks/ApplicationInsights/RequestTests.cs

```diff
@@ -23,6 +23,81 @@ public RequestTests(ITestOutputHelper outputWriter) : base(outputWriter)
         {
         }
 
+        [Fact]
+        public async Task LogRequest_SinksToApplicationInsightsWithoutOperationName_ResultsInRequestTelemetry()
+        {
+            // Arrange
+            HttpMethod httpMethod = GenerateHttpMethod();
+            var requestUri = new Uri(BogusGenerator.Internet.UrlWithPath());
+            HttpRequest request = CreateStubRequest(httpMethod, requestUri.Scheme, requestUri.Host, requestUri.AbsolutePath);
+
+            using (ILoggerFactory loggerFactory = CreateLoggerFactory())
+            {
+                ILogger logger = loggerFactory.CreateLogger<ApplicationInsightsSinkTests>();
+
+                var statusCode = BogusGenerator.PickRandom<HttpStatusCode>();
+                HttpResponse response = CreateStubResponse(statusCode);
+
+                TimeSpan duration = BogusGenerator.Date.Timespan();
+                Dictionary<string, object> telemetryContext = CreateTestTelemetryContext();
+
+                // Act
+                logger.LogRequest(request, response, duration, telemetryContext);
+            }
+
+            // Assert
+            using (ApplicationInsightsDataClient client = CreateApplicationInsightsClient())
+            {
+                await RetryAssertUntilTelemetryShouldBeAvailableAsync(async () =>
+                {
+                    EventsResults<EventsRequestResult> results = await client.Events.GetRequestEventsAsync(ApplicationId, filter: OnlyLastHourFilter);
+                    Assert.NotEmpty(results.Value);
+                    Assert.Contains(results.Value, result => result.Request.Url == $"{requestUri.Scheme}://{requestUri.Host}{requestUri.AbsolutePath}");
+                    Assert.Contains(results.Value, result => !String.IsNullOrEmpty(result.Operation.Name));
+                });
+            }
+
+            VerifyLogEventProperties(requestUri);
+        }
+
+        [Fact]
+        public async Task LogRequest_SinksToApplicationInsightsWithOperationName_ResultsInRequestTelemetry()
```

Comment:
Great stuff! Shows how we can pass along an operation name, now. Since the other dependency logging extensions have been altered to include a dependency name, we should also test for those. (unless you want to split the other dependencies in another PR).
Normally, we have for every logging extension an integration test, so in some cases you could only add an extra assertion for the dependency name, without having to write a dedicated test like these ones.
Thanks again!

Reply:
I added the dependency tests to check for the dependency name. Added some tests as well, let me know if tests are still missing 👍🏻

Reply:
I think there' all there. Both unit and integration tests updated. Great!